docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  web:
+    build: .
+    command: bash -c "cd parking_manager && python manage.py makemigrations && python manage.py migrate && python manage.py loaddata parking_app/fixtures/*.json && python manage.py runserver 0.0.0.0:8000"
```

Comment:
I was thinking in placing all the commands into the `dockerfile` and in the `docker-compose.yml` just use it like this:

```yml
services:
  postgres:
    build:
      dockerfile: Dockerfile
```

What do you think?

Reply:
hi, I would say we can do it in many ways ... I learned that from the documentation and I think it's just fine..